### PR TITLE
[JS] update API docs for cookies section

### PR DESCRIPTION
**Pull request by manoj9788**
- [X ] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

### CODE REVIEW

File: javascript/node/selenium-webdriver/lib/webdriver.js [hunk 1]
```diff
@@ -1170,6 +1170,11 @@ class Options {
   }
 
   /**
+   * __Sample Usage:__
+   * 
+   * //delete all the cookies
+   * driver.manage().deleteAllCookies();
+   * 
    * Schedules a command to delete all cookies visible to the current page.
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/lib/webdriver.js]: Method description should be before the sample usage.

File: javascript/node/selenium-webdriver/lib/webdriver.js [hunk 2]
```diff
@@ -1170,6 +1170,11 @@ class Options {
   }
 
   /**
+   * __Sample Usage:__
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/lib/webdriver.js]: Do we really need sample usage? This seems straightforward

Comment (manoj9788) [inline javascript/node/selenium-webdriver/lib/webdriver.js]: Its a nice to have, so added. Like sample usage for all  the API.

File: javascript/node/selenium-webdriver/lib/webdriver.js [hunk 3]
```diff
@@ -1171,6 +1171,12 @@ class Options {
 
   /**
    * Schedules a command to delete all cookies visible to the current page.
+   * 
+   * __Sample Usage:__
+   * 
+   * //delete all the cookies
```
Comment (jleyba) [inline javascript/node/selenium-webdriver/lib/webdriver.js]: These lines need to be indented 4 spaces past the * to be formatted as a code block in the generated docs. To check your formatting, run `node javascript/node/gendocs.js`

Output will be generated in `build/javascript/node/selenium-webdriver-docs`

Comment (manoj9788) [inline javascript/node/selenium-webdriver/lib/webdriver.js]: I will check.

**Pull request closed without merging**